website/docusaurus.config.js

```diff
@@ -28,11 +28,11 @@ module.exports = {
   themeConfig: {
     metadata: [
       {name: 'description', content: 'Apache Pinot'},
-      {name: 'description', content: 'Real-time Analytics'},
-      {name: 'description', content: 'User-facing Analytics'},
+      {name: 'description', content: 'real-time Analytics'},
+      {name: 'description', content: 'user-facing Analytics'},
       {name: 'description', content: 'user facing analytics'},
```

Comment:
Should it all be capitalized?

Reply:
Good catch. Consistent title case is best!